string token
string timestamp
float64[] ego2global
int32[] labels
int32[] ids
float64[] bboxes
string[] states